### PR TITLE
Re-organize readme, add two tutorials

### PR DESCRIPTION
**Pull request by SamitHuang**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: docs/cn/tutorials/training_on_openi.md
Comment (zhtmike) [inline docs/cn/tutorials/training_on_openi.md:1]: openi上启智集群目前无法用NPU集群，只能用智算网络NPU集群。先版本无法在智算网络上跑，需要适配，我会另开 PR适配

Comment (zhtmike) [reply docs/cn/tutorials/training_on_openi.md:1]: #395

Comment (SamitHuang) [reply docs/cn/tutorials/training_on_openi.md:1]: 改了之后能跟modelarts平台适配吗？

Comment (zhtmike) [reply docs/cn/tutorials/training_on_openi.md:1]: 不清楚智算是否跟modelart使用同一套接口

**Review by wtomin — APPROVED** [review]

File: docs/en/tutorials/advanced_train.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Gradient clipping is a method to address gradient explosion/overflow problem and to
+stabilize model convergence.
+
+To enable it, please set `train.ema` as True and optionally adjust the norm value by `train.clip_norm`.
+
+
+#### EMA
+
+Exponential Moving Average (EMA) can be viewed as a model ensemble method that smooth the model weights.
```
Comment (hadipash) [inline docs/en/tutorials/advanced_train.md:38]: that smooth the model weights across multiple training steps?

File: docs/cn/tutorials/training_on_openi.md
Comment (SamitHuang) [reply docs/cn/tutorials/training_on_openi.md:1]: 之前是跟modelart应该是一套接口，为了后续aicc能正常使用，应该保持兼容。

**Review by colawyee — APPROVED** [review]

**Pull request merged**